vip-jetpack/vip-jetpack.php

```diff
@@ -9,6 +9,12 @@
  * License: GPL2+
  */
 
+define( 'VIP_GO_JETPACK_SYNC_MAX_QUEUE_SIZE_LOWER_LIMIT', 10000 );
+define( 'VIP_GO_JETPACK_SYNC_MAX_QUEUE_SIZE_UPPER_LIMIT', 50000 );
+
+define( 'VIP_GO_JETPACK_SYNC_MAX_QUEUE_LAG_LOWER_LIMIT', 2 * HOUR_IN_SECONDS );
+define( 'VIP_GO_JETPACK_SYNC_MAX_QUEUE_LAG_UPPER_LIMIT', 6 * HOUR_IN_SECONDS );
```

Comment:
Should we wrap these in a `defined` check? Would allow customizing on a per-site basis, if needed.

Reply:
I see these more as hard limits - the sites can still be customized via the option, but these are the "bumpers" that sites can't go above or below (too big or too small can break the site). We can allow changing these though if you think it'll be valuable.

Reply:
Oh, you're right :)

I thought the goal here was to set higher defaults across VIP, not just better limits? Or is that coming as a separate PR?

Reply:
Never mind -- my brain is clearly not working. I see the min bumps those up :)

Let's add some comments to the constant definitions noting why they're set to those specific values.

Reply:
👍 added explanatory comments to the constants in da6da27f